Cargo.toml

```diff
@@ -42,6 +42,7 @@ log = "0.4"
 env_logger = "0.7"
 url = "2"
 fixedbitset = "0.3"
+json = "0.12.4"
```

Comment:
Can we just use serde_json as in the other dezoomers ? This gives simpler code, better json structure validation, better error handling, and better error messages.

Reply:
will be done.